aspnetcore/signalr/version-differencs.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Differences between SignalR and ASP.NET Core SignalR
+author: rachelappel
+description: Differences between SignalR and ASP.NET Core SignalR 
+manager: wpickett
```

Comment:
Remove the following metadata fields:
- `manager`
- `ms.prod`
- `ms.technology`
- `ms.topic`

These will pull from *docfx.json*.